src/index.js

```diff
@@ -14,7 +14,13 @@ import Game from '../src/Game.js';
 import DOMupdates from '../src/DOMupdates.js';
 
 //Fetch data
-let game = new Game(data);
+let data;
+let game;
```

Comment:
You mightttt run into issues if you are relying on these being accessible because they are global variables. Your app will work just fine but your tests might get sad about it. Wherever you want access to this data, you’ll want to pass it through as an argument to the methods that need it. *Or* you could try setting a global variable in your test file like `global.data = yourMockData`. Not sure off the top of my head if that will work but something to try in case you find you run into problems.